core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -313,12 +391,12 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
       }
       val startingCount = prewarmStartingPool.count(p => p._2._1 == kind && p._2._2 == memory)
       val containerCount = currentCount + startingCount
-      if (containerCount < config.count) {
+      if (containerCount < config.initialCount) {
```

Comment:
`config.initialCount` is being used multiple times.
How about introducing a local variable for this?

Reply:
Already added